check-grammars-crates.sh

```diff
@@ -107,8 +107,11 @@ if [ "$(ls -A $COMPARE)" ]; then
     for METRIC in "${METRICS[@]}"
     do
 
-        PREFIX_METRIC="\.$METRIC"
-        FILES=`grep -r -i -l $PREFIX_METRIC $COMPARE | head -$MT_THRESHOLD`
+        # Find metrics as whole words
+        MATCH_PATTERN="\b\.$METRIC\b"
+        # Exclude these patterns for metrics
```

Comment:
Why should this pattern be excluded?

Reply:
Because in this way we exclude files containing the name of a metric inside minimal tests code. Just an example:
The `nom` metric is different, so we have an `.html` file. Inside this file we have a function called `blank()`. When minimal tests are divided according to metrics, we have the `nom` html file inside the blank directory, but the `blank` metric wasn't modified at all

Reply:
Can you expand the comment in the file with what you just said?

Reply:
Sure :)